.github/workflows/docs.yml

```diff
@@ -16,11 +15,16 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
 
-      - name: Setup python
+      - name: Set up Python
         uses: actions/setup-python@e797f83bcb11b83ae66e0230d6156d7c80228e7c # v6.0.0
         with:
-          python-version: '3.13'
-          cache: 'pip'
+          python-version-file: "pyproject.toml"
+
+      - name: Install uv
+        uses: astral-sh/setup-uv@d0cc045d04ccac9d8b7881df0226f9e82c39688e # v6.8.0
+        with:
+          enable-cache: true
+          version: "0.8.22"
```

Comment:
I'm a bit hesitant to pin this. If it were to auto-update each week (dependabot) this would be fine, but I'm pretty sure we'd rarely ever manually update this. I would be in favor of using the latest & greatest `uv` version and risk it breaking. We've been doing this for about a year now and have not encountered any issues.

Reply:
Sure, using a repo variable sounds better. Let's do that. I've added this:

<img width="353" height="154" alt="image" src="https://github.com/user-attachments/assets/6fdb0c3c-1b0f-456f-8dbd-fdb850f7a716" />